src/hooks/chats/privateChats/useOnlineUsersToDisplay.ts

```diff
@@ -0,0 +1,16 @@
+import { useRecentWorldUsers } from "../../users";
+import { useUser } from "../../useUser";
+import { useMemo } from "react";
```

Comment:
please, sort the imports

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/3660c406551764e6c60917460a79a253b51c4a67